modules/url-check.js

```diff
@@ -1,8 +1,10 @@
 const { red } = require('kleur')
 
+const httpsRegex = /https?:\/\//
```

Comment:
```suggestion
const httpsRegex = /^https?:\/\//
```

Reply:
And please move it into the exported body. See cli-help.js for example. Only module / dependency imports should be outside.